packages/main/src/types/BadgeSize.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Defines badge size types.
```

Comment:
"size types" is not the best wording. Maybe:
Defines badge size options.
or
Predefined sizes for the badge.

Reply:
"Predefined sizes for the badge". sounds good to me